### PR TITLE
dssp 3.1.4 (new formula)

### PR DESCRIPTION
**Pull request by YoshitakaMo**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

This is a new formula of dssp, which is related to https://github.com/brewsci/homebrew-bio/pull/1189.

### CODE REVIEW

File: Formula/dssp.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  homepage "https://github.com/cmbi/dssp"
+  url "https://github.com/cmbi/dssp/archive/3.1.4.tar.gz"
+  sha256 "496282b4b5defc55d111190ab9f1b615a9574a2f090e7cf5444521c747b272d4"
+
+  bottle do
+    root_url "https://linuxbrew.bintray.com/bottles-bio"
+    cellar :any
+    sha256 "c0833e3e3db899b28dd9515e1d3ffb791528d2934d25226dbf4eaf0f0428b34f" => :sierra
+    sha256 "890f4a0285987df828d271ad333b597181358c75b65026d94dccf3d7c6d49531" => :x86_64_linux
+  end
```
Comment (sjackman) [inline Formula/dssp.rb:14]:
```suggestion
```
You can delete the bottle block. It's added by CI after the bottles are built.

File: Formula/dssp.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "boost"
```
Comment (sjackman) [inline Formula/dssp.rb:18]: Is there run-time linkage to `boost`? What's the output of `brew linkage dssp`?

**Review by sjackman — CHANGES_REQUESTED** [review]

File: Formula/dssp.rb (hunk 2 above)
Comment (YoshitakaMo) [reply Formula/dssp.rb:11]: Here is the output of `brew linkage dssp`.

```
$ brew linkage dssp
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libc++.1.dylib
Homebrew libraries:
  /usr/local/opt/boost/lib/libboost_date_time-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_filesystem.dylib (boost)
  /usr/local/opt/boost/lib/libboost_iostreams-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_program_options-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_system.dylib (boost)
  /usr/local/opt/boost/lib/libboost_thread-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_unit_test_framework-mt.dylib (boost)
```

But another issue has happened on the build-linux-bottles.

**Review by sjackman — APPROVED** [review]

**Comment by sjackman** [comment]
Thanks again, @YoshitakaMo ! 🏆

**Pull request merged**